apps/web/components/molecules/opinionItem/opinionItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, Heading, Text } from 'ui';
+import Image from 'next/image';
+import clsx from 'clsx';
+import type { Opinion } from '../../../types/types';
+
+type OpinionItemProps = Opinion;
+
+export const OpinionItem = ({
+	author,
+	avatar,
+	rating,
+	comment,
+}: OpinionItemProps) => {
+	return (
+		<div className="p-2">
+			<Card tag="div">
+				<div className="flex flex-col md:flex-row justify-center items-center">
+					<Image
+						src={avatar}
+						alt=""
+						width={1000}
+						height={1000}
+						className="w-full h-full rounded-full w-24 h-24 object-cover"
+						aria-hidden="true"
+					/>
+					<div className="md:pl-5">
+						<Heading tag="h3" size="large" className="font-bold">
+							{author}
+						</Heading>
+						<div className="flex items-center">
+							{Array.from({ length: 5 }, (_, i) => {
```

Comment:
Would extract this 5 to some constant. Also I'd create an array with some meaningful name and put it outside of the component as it is static.